docs/source/model_design/vivarium_model_components/causes/index.rst

```diff
@@ -827,5 +827,43 @@ Morbidity Impacts
+
+YLD Uncertainty
+^^^^^^^^^^^^^^^
+
+The uncertainty ranges reported around YLDs incorporate uncertainty in prevalence and uncertainty in the DW. To do this, we take the 1000 samples of comorbidity-corrected YLDs and 1000 samples of the DW to generate 1000 samples of the YLD distribution. We assume no correlation in the uncertainty in prevalence and DWs. The 95% uncertainty interval is reported as the 25th and 975th values of the distribution. UIs for YLDs at different points in time (1990, 1995, 2000, 2005, 2010, and 2016) for a given disease or sequela are correlated because of the shared uncertainty in the DW. For this reason, changes in YLDs over time can be significant even if the UIs of the two estimates of YLDs largely overlap because significance is determined by the uncertainty around the prevalence estimates.
+
+
+Residual YLDs
+^^^^^^^^^^^^^
+
+For less common diseases and their sequelae, GBD may not currently estimate disease prevalence and YLDs, and have thus been included in residual categories. For these residual categories, we estimate YLDs by multiplying the residual YLL estimates by the ratio of YLDs to YLLs from the estimates of Level 3 causes in the same disease category that were explicitly modelled. This scaling is done for each country-sex-year. 
+
+Incidence- vs. Prevalence-Based YLDs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*Incidence-based YLDs* provide a measure of the disease burden experienced by an individual over the course of their lifetime. Incidence-based YLDs are calculated by adding together YLD 1 (i.e., the product of the DW associated with disease 1 and the time between onset and survival) and YLD 2 (i.e., the product of the DW associated with disease 2 and the time between onset and survival). 
+
+
+*Prevalence-based YLDs*, on the other hand, are what we tend to use in Vivarium models. Prevalence-based YLDs reflect the burden of disease in the year of incidence. 
+
+.. list-table:: Incidence vs. Prevalence-based YLDs (`Kim et al., 2022 <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8841194/>`_)
+   :widths: 30 30 30 
+   :header-rows: 1
+
+   * - Measure
+     - Advantages
+     - Disadvantages
+   * - Incidence-based YLDs
+     - YLDs and YLLs are measured consistently; Provide a more valid data source for policies or interventions focused on disease prevention.
```

Comment:
With incidence based, do we risk overestimating this? Since we assume you will lose all of the possible YLDs without considering other possible comorbidity impacts. For example, if a very comorbid person had a heart attack, would we still say you get 10 YLDs from this event, regardless of if they also have diabetes, stroke, etc etc which would decrease the YLDs in practice. 

This might already by considered in the value for YLDs we assign on that time step though.

Reply:
I think this is why one of the advantages of prevalence-based YLDs below is "Incorporation of comorbidities is easier."

If using incidence-based, we could at least consider the chronic/incurable comorbidities already present, but there would be no easy way to adjust the YLDs to account for e.g. a risk factor that is not causing any disability right now but means they are more likely to have more comorbidities later in life.